orange-datamodel/src/main/java/com/hartwig/hmftools/datamodel/linx/LinxRecord.java

```diff
@@ -30,6 +30,9 @@ public interface LinxRecord
     @NotNull
     List<LinxFusion> additionalSuspectSomaticFusions();
 
+    @NotNull
+    List<LinxFusion> additionalViableFusions();
```

Comment:
I would recommend to rename to `additionalViableSomaticFusions()` to make it clear it is somatic only (in theory there could be germline fusions as well though not sure in practice... but in any case every property in linx currently has somatic or germline in their name